cypress/tests/scroll-to-contest.js

```diff
@@ -10,9 +6,9 @@ describe('Review Page', () => {
     })
     cy.contains('Submit').click({ force: true })
     cy.contains('Get Started').click()
-    cy.wrap(clickThoughPages).each(() => {
+    while (!cy.contains('Review Ballot')) {
```

Comment:
I'm ok with hard-coding an array with fixed length because a user will click through N pages to get to the review screen.

Reply:
oh cool. That helps.